dashboard/test/models/level_source_test.rb

```diff
@@ -39,31 +14,4 @@ class LevelSourceTest < ActiveSupport::TestCase
     refute level_source.valid?
     assert_equal ['Data is invalid'], level_source.errors.full_messages
   end
-
-  test "should validate level_source factory for md5" do
```

Comment:
Note that this test really should not be deleted as part of this PR, rather it should be deleted as part of #13578. But I'm deleting it here too to prevent merge conflicts.